cmd/check.go

```diff
@@ -45,7 +44,7 @@ Flags:
 			return types.Error{Msg: "Unable to create OpCap client."}
 		}
 		var packageManifestList pkgserverv1.PackageManifestList
-		err = psc.ListPackageManifests(context.TODO(), &packageManifestList, checkflags)
+		err = psc.ListPackageManifests(context.TODO(), &packageManifestList, checkflags.CatalogSource, checkflags.FilterPackages)
```

Comment:
Hi @yashoza19 , I guess this can be part of the list packages issue #185 . It will definitely interfere with that. This piece should actually disappear after the other issue is solved. What do you think?

Reply:
Yes, agreed @acmenezes . I will update that issue with more information.